include/SPTPersistentCache/SPTPersistentCache.h

```diff
@@ -195,8 +195,12 @@ typedef NSString * _Nonnull(^SPTPersistentCacheChooseKeyCallback)(NSArray<NSStri
 /**
  * @brief Removes data for keys unconditionally even if expired.
  * @param keys The keys corresponding to the data to remove.
+ * @param callback May be nil if not interested in result.
+ * @param queue Queue on which to run the callback. If callback is nil this is ignored otherwise mustn't be nil.
  */
-- (void)removeDataForKeys:(NSArray<NSString *> *)keys;
+- (void)removeDataForKeys:(NSArray<NSString *> *)keys
```

Comment:
Could we make this an additive change instead, keeping the old method around so we don’t break the current API?